azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingViewModel.kt

```diff
@@ -85,6 +92,18 @@ internal class CallingViewModel(
         confirmLeaveOverlayViewModel.requestExitConfirmation()
     }
 
+    fun requestToggleHold() {
+        if (store.getCurrentState().callState.callingStatus == CallingStatus.LOCAL_HOLD) {
```

Comment:
I feel it's better to put the state check `store.getCurrentState().callState.callingStatus == CallingStatus....` to callingMiddlewareHandler

Reply:
Not sure I follow. The middleware just responds to actions and filters/mod. If current state is local_hold we resume, and if current state is connected, we hold. 

However, this was just scaffolding code for when I had a toggle button, it's been deleted now since we don't have a hold button on the Calling screen.